### PR TITLE
[AIRFLOW-6948] Remove ASCII Airflow from version command

### PR DESCRIPTION
**Pull request by turbaszek**
Before:
```
root@3f957e731266:/opt/airflow# airflow version
[2020-02-27 16:09:09,092] {settings.py:177} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=902
  ____________       _____________
 ____    |__( )_________  __/__  /________      __
____  /| |_  /__  ___/_  /_ __  /_  __ \_ | /| / /
___  ___ |  / _  /   _  __/ _  / / /_/ /_ |/ |/ /
 _/_/  |_/_/  /_/    /_/    /_/  \____/____/|__/  v2.0.0.dev0
```

after:

```
root@3f957e731266:/opt/airflow# airflow version
v2.0.0.dev0
```

---
Issue link: [AIRFLOW-6948](https://issues.apache.org/jira/browse/AIRFLOW-6948)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: airflow/settings.py [hunk 1]
```diff
@@ -173,8 +173,8 @@ def configure_orm(disable_connection_pool=False):
         # https://docs.sqlalchemy.org/en/13/core/pooling.html#disconnect-handling-pessimistic
         pool_pre_ping = conf.getboolean('core', 'SQL_ALCHEMY_POOL_PRE_PING', fallback=True)
 
-        log.info("settings.configure_orm(): Using pool settings. pool_size={}, max_overflow={}, "
-                 "pool_recycle={}, pid={}".format(pool_size, max_overflow, pool_recycle, os.getpid()))
+        log.debug("settings.configure_orm(): Using pool settings. pool_size=%d, max_overflow=%d, "
+                  "pool_recycle=%d, pid=%d", pool_size, max_overflow, pool_recycle, os.getpid())
```
Comment (feluelle) [inline airflow/settings.py:177]: I think this should be a separate PR. This log will not only be displayed when printing airflow version.

Comment (turbaszek) [reply airflow/settings.py:177]: You are right, however, I am not sure if this log information is so crucial that it deserves separate PR? @feluelle @mik-laj

**Review by mik-laj — APPROVED** [review]

**Review by kaxil — APPROVED** [review]

File: airflow/cli/commands/version_command.py [hunk 2]
```diff
@@ -16,9 +16,8 @@
 # under the License.
 """Version command"""
 import airflow
-from airflow import settings
 
 
 def version(args):
     """Displays Airflow version at the command line"""
-    print(settings.HEADER + "  v" + airflow.__version__)
+    print(f"v{airflow.__version__}")
```
Comment (kaxil) [inline airflow/cli/commands/version_command.py]: We should probably remove `v` too.

Comment (kaxil) [inline airflow/cli/commands/version_command.py]:
```suggestion
    print(f"{airflow.__version__}")
```

File: airflow/settings.py (hunk 1 above)
Comment (zhongjiajie) [reply airflow/settings.py:177]: I think it ok, just a small change, unless we create a PR change all log and and some static test for it

**Review by feluelle — APPROVED** [review]
Fine! :)

File: airflow/cli/commands/version_command.py [hunk 3]
```diff
@@ -16,9 +16,8 @@
 # under the License.
 """Version command"""
 import airflow
-from airflow import settings
 
 
 def version(args):
     """Displays Airflow version at the command line"""
-    print(settings.HEADER + "  v" + airflow.__version__)
+    print(f"{airflow.__version__}")
```
Comment (feluelle) [inline airflow/cli/commands/version_command.py]:
```suggestion
    print(airflow.__version__)
```

Wait a second.. why are we now still using f string if it is only one variable without any extra string attached to it? 😁

Comment (zhongjiajie) [reply airflow/cli/commands/version_command.py]: Good catch！

Comment (turbaszek) [reply airflow/cli/commands/version_command.py]: Fixed 😅

Comment (kaxil) [reply airflow/cli/commands/version_command.py]: 😅

**Review by kaxil — APPROVED** [review]
